test/rules-engine/index.test.js

```diff
@@ -0,0 +1,30 @@
+describe('Rules engine help methods test', () => {
+  let rulesEngine
+  const rulesTestData = require('./test-data/rules-index.json')
+
+  beforeAll(async () => {
+    rulesEngine = require('../../server/rules-engine')
```

Comment:
I normally would just declare rulesEngine as a const at the top of the describe block, then this function can be disposed of. If you prefer it as it is, I think the async needs to go as there's no `await` in the function.

Reply:
Changed